deploy/database/drop_databases.sql

```diff
@@ -1,2 +0,0 @@
-DROP DATABASE IF EXISTS buttonmen;
-DROP DATABASE IF EXISTS buttonmen_test;
```

Comment:
It's not orphaned --- i never used it, and you introduced it recently for use by AMP, which is why i was surprised that you removed it.  But, anyway, i have no objection.